Clarify WillPopScope disables predictive back (#10928)
flutter/website: src/content/release/breaking-changes/android-predictive-back.md

### PopScope

> The `PopScope` class directly replaces `WillPopScope` in order to enabled

Reviewer: ```suggestion
The `PopScope` class directly replaces `WillPopScope` in order to enable
```

     for migrating Android apps to support predictive back.
  1. Make sure you're using version `3.14.0-7.0.pre`
     of Flutter or greater.

>   1. Make sure your Flutter app does not use the
>      WillPopScope widget. Using it will disable
>      predictive back. Use PopScope instead if
>      needed.

Reviewer: ```suggestion
  1. Make sure your Flutter app doesn't use the
     `WillPopScope` widget. Using it disables
     predictive back. If needed, use `PopScope` instead.
```
